packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts

```diff
@@ -14,14 +14,44 @@ export class AtomicCommerceSearchBoxLocators {
     return this.page.getByRole('combobox', {name: 'Search'});
   }
 
+  get clearButton() {
+    return this.page.getByLabel('Clear', {exact: true});
+  }
+
+  get ariaLive() {
+    return this.page.locator('div[id*="aria-live-"][id$="-search-box"]');
+  }
+
+  get clearRecentQueriesButton() {
+    return this.page.getByLabel('Clear recent searches.');
+  }
+
+  get numberOfQueries() {
```

Comment:
This is a bit flawed since it check the attribute on the search box, not the actual number of suggestion shown to the end user. ie: the attribute/property could be bugged and the selector would be invalid.

How about trying to `getByRole('button')` with some filter on the button attributes to snipe only suggestions ?

Reply:
I'm using this to validate that the aria-live message contains the smallest value between the count of suggestions+recent queries and the maximum number of suggestions as set on the search box.